### PR TITLE
hw7

### PR DESCRIPTION
**Pull request by un-install**

### CODE REVIEW

File: src/main/java/homework5/Fruits.java [hunk 1]
```diff
@@ -47,10 +51,19 @@ public void setVitamins (List<Vitamins> vitamins) {
         this.vitamins = vitamins;
     }
 
-    public void setPrice(int price) {
+    public void setPrice(int price) throws ArithmeticException{
+        if (price <= 0) {
+            throw new ArithmeticException("Price can be only positive");
```
Comment (SergeyPasko) [inline src/main/java/homework5/Fruits.java]: Good

File: src/main/java/homework7/Main.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static homework5.FruitsEnum.*;
+import static homework6.Vitamins.*;
+
+public class Main {
+    public static void main(String[] args) {
+        homework6.Main main = new homework6.Main();
```
Comment (SergeyPasko) [inline src/main/java/homework7/Main.java]: What sence?

Comment (SergeyPasko) [reply src/main/java/homework7/Main.java]: It isn't need

Comment (un-install) [reply src/main/java/homework7/Main.java]: In homework6.Main is a private method fillFruitList. This instance is here to use it

Comment (un-install) [reply src/main/java/homework7/Main.java]: via reflection

Comment (un-install) [reply src/main/java/homework7/Main.java]: I redone with Class.forName()

File: src/main/java/homework7/Main.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            }
+            ).collect(Collectors.toList());
+
+            //output
+            fruits.forEach(System.out::println);
+            System.out.println();
+            fruitsClone.forEach(System.out::println);
+
+            //get private Constructor using reflex API
+            Constructor constructor = Class.forName(Animal.class.getName()).getDeclaredConstructor();
```
Comment (Cherkashyn) [inline src/main/java/homework7/Main.java]: Too heavy. You can use Animal.class.getDeclaredConstructor

File: src/main/java/homework7/Main.java (hunk 2 above)
Comment (un-install) [reply src/main/java/homework7/Main.java]: but it dont work

**Comment by un-install** [comment]
.

**Pull request merged**